include/quicly.h

```diff
@@ -470,16 +493,25 @@ struct _st_quicly_conn_public_t {
         /**
          * stateless reset token corresponding to the CID
          */
-        struct {
-            uint8_t *token;
-            uint8_t _buf[QUICLY_STATELESS_RESET_TOKEN_LEN];
-        } stateless_reset;
+        struct st_quicly_stateless_reset_t stateless_reset;
```

Comment:
I do not think we need this generalization. The reason we have `*token` and `_buf` is because we might not have stateless reset tokens for the first CID. That's unique to the first CID, and therefore we do not need to generalize it.

Reply:
Sounds good. Done.